res/css/views/rooms/_IRCLayout.scss

```diff
@@ -15,115 +15,141 @@ limitations under the License.
 */
 
 $icon-width: 14px;
-$right-padding: 5px;
+$right-padding: 5px; // TODO: Use a spacing variable
 $irc-line-height: $font-18px;
 
 .mx_IRCLayout {
     --name-width: 70px;
+    --IRCLayout_EventTile_line_child-padding-block: 1px;
 
     line-height: $irc-line-height !important;
 
-    .mx_EventTile {
-        // timestamps are links which shouldn't be underlined
-        > a {
-            text-decoration: none;
-            min-width: $MessageTimestamp_width;
-        }
-
+    .mx_EventTile[data-layout=irc] {
```

Comment:
[`:not()` can increase the specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/:not#description) of a rule, therefore including `.mx_EventTile` by `.mx_IRCLayout` should conflict with `:not([data-layout=bubble])` and has not been quite enough without `[data-layout=irc]`.